index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Wasyihun-portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div>
+        <a class="My_logo">MyLogo</a>
+      </div>
+      <nav class="menu_icon">
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+      </nav>
+    </header>
+    <section class="Body_content">
+      <div class="Body_content_title">
+        <h1>
+          I’m Wasyihun<br />
+          Glad to see<br />
+          you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h2>LET’S CONNECT</h2>
+        <ul>
+          <li><img src="image/Twittwer_vector.png" alt="" /></li>
+          <li><img src="image/Linkedin_icon.png" alt="" /></li>
+          <li><img src="image/M_vector.png" alt="" /></li>
+          <li><img src="image/Github_vector.png" alt="" /></li>
+          <li><img src="image/Pieace_vector.png" alt="" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="Works">
+      <main class="Works_main">
+        <img class="big_pictures" src="image/Picture_1.png" alt="" />
+        <h2>Tonic</h2>
```

Comment:
The `font-size` of the card header is `32px` as per Figma design but the one used in your design is smaller. Kindy take a look at all the `h2` `font-size` of cards.